src/sentry/static/sentry/app/bootstrap.jsx

```diff
@@ -63,13 +63,23 @@ if (window.__initialData) {
 }
 
 // APM -------------------------------------------------------------
+// A list of enabled customers to sample apm from
+const enabledCustomers = ['sentry', 'riotgames', 'squareup'];
```

Comment:
Before it seemed like having 'sentry' in the email qualified a user for APM not sure if we want to use regex or some more specific substring to match on ('sentry.io', 'riotgames.com') or if this is sufficient